apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -529,6 +529,13 @@ type RolloutStrategy struct {
 	// +kubebuilder:validation:Enum=RollingUpdate;External
 	Type RolloutStrategyType `json:"type,omitempty"`
 
```

Comment:
why does this option not require 

ResourceSnapshotCreationPolicy ResourceSnapshotCreationPolicyType `json:"resourceSnapshotCreationPolicy,omitempty"`